docs/topics/mta-web-adding-applications.adoc

```diff
@@ -10,12 +10,12 @@ You can add an application to the *Application Inventory* for assessment and ana
 
 [TIP]
 ====
-Before creating an application, it is helpful to set up business services, check tags and tag types, and create additions as needed.
+Before creating an application, it is helpful to set up business services, check tags and categories, and create additions as needed.
```

Comment:
can you please replace with **Tag categories**

Reply:
Done